src/FakeItEasy.dotnet/project.json

```diff
@@ -1,24 +1,24 @@
 {
   "dependencies": {
     "Castle.Core": "4.0.0-alpha001",
-    "Microsoft.Extensions.DependencyModel": "1.0.0-rc2-final",
-    "Newtonsoft.Json": "8.0.4-beta1",
-    "System.Collections.Concurrent": "4.0.12-rc2-24027",
-    "System.Console": "4.0.0-rc2-24027",
-    "System.Diagnostics.Tools": "4.0.1-rc2-24027",
-    "System.Diagnostics.Debug": "4.0.11-rc2-24027",
-    "System.IO.FileSystem": "4.0.1-rc2-24027",
-    "System.IO.FileSystem.Primitives": "4.0.1-rc2-24027",
-    "System.Linq.Expressions": "4.0.11-rc2-24027",
-    "System.Reflection": "4.1.0-rc2-24027",
-    "System.Reflection.Emit": "4.0.1-rc2-24027",
-    "System.Reflection.Extensions": "4.0.1-rc2-24027",
-    "System.Reflection.TypeExtensions": "4.1.0-rc2-24027",
-    "System.Runtime.Loader": "4.0.0-rc2-24027"
+    "Microsoft.Extensions.DependencyModel": "1.0.0",
+    "Newtonsoft.Json": "9.0.1",
+    "System.Collections.Concurrent": "4.0.12",
+    "System.Console": "4.0.0",
+    "System.Diagnostics.Tools": "4.0.1",
+    "System.Diagnostics.Debug": "4.0.11",
+    "System.IO.FileSystem": "4.0.1",
+    "System.IO.FileSystem.Primitives": "4.0.1",
+    "System.Linq.Expressions": "4.1.0",
+    "System.Reflection": "4.1.0",
+    "System.Reflection.Emit": "4.0.1",
+    "System.Reflection.Extensions": "4.0.1",
+    "System.Reflection.TypeExtensions": "4.1.0",
+    "System.Runtime.Loader": "4.0.0"
   },
 
   "frameworks": {
-    "netstandard1.5": {
+    "netstandard1.6": {
       "imports": [ "dotnet5.4" ]

```

Comment:
Is the `imports` still necessary? None of the dependencies seem to target `dotnet5.4` (I didn't check them all, but `Newtonsoft.Json` targets `netstandard1.0`, and the `System.*` and `Microsoft.*` I checked target `netstandard 1.6`)

(of course, it's also possible that I completely misunderstand what `imports` does :wink:)